pages/your_role/principal_investigator.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Principal Investigator (PI)
+description: Software development and quality guidance for Principal Investigators (PIs)
+contributors: ["Aleksandra Nenadic"]
+page_id: principal_investigator
+---
+
+## Introduction
+
+As a Principal Investigator (PI), you may have recently secured (or are in the process of applying for) project funding 
+that includes software development. Increasingly, funders are requiring a Software Management Plan (SMP), 
+encouraging PIs to proactively consider key aspects of software development and long-term sustainability from the very 
+start of the project.  
+
+## Software development & quality responsibilities, challenges & tasks
+
+As a Principal Investigator (PI), your responsibilities in relation to software development and quality may include:
+
+- Developing a Software Management Strategy - create a clear plan for how software will be developed, maintained, and 
+shared throughout the project. This includes defining sustainability goals, allocating resources, and budgeting appropriately—often formalised in a Software Management Plan required by many funders.
+- Incorporating the FAIR research software principles in the software development life-cycle — ensuring that your software 
+is Findable, Accessible, Interoperable, and Reusable to support transparency, collaboration, and long-term impact.
+European Commission (Horizon Europe) and many national funding bodies strongly encourage adherence to the FAIR principles for both data and software. 
+- Establishing roles and responsibilities - identify and assign software-related roles within your team 
+(e.g., developer, maintainer, reviewer, documenter, tester, early adopter) to foster accountability, collaboration, and efficient workflows across the project lifecycle.
+- Addressing ethical, environmental, and legal considerations early:
+  - Ethical: promote responsible use of software and data, ensure transparency, and consider contributing improvements or reusable components back to the community.
+  - Environmental: be aware of the energy demands and carbon footprint of your software—opt for efficient algorithms, cloud providers using renewable energy, or local computing where feasible.
```

Comment:
Perhaps don't say too much e.g. local computing might not always be great energy efficiency wise - might be worth linking to the task page also - https://everse.software/RSQKit/improving_environmental_sustainability#how-to-monitor-and-reduce-the-environmental-impact-of-my-software